core/src/test/java/io/questdb/test/cairo/TimestampFinderTest.java

```diff
@@ -49,20 +49,20 @@ public void setUp() {
 
     @Test
     public void testFuzzAllDuplicates() throws Exception {
-        testFuzz(1000, 1000);
+        testFuzz(1000);
     }
 
     @Test
     public void testFuzzFewDuplicates() throws Exception {
-        testFuzz(1000, 1);
+        testFuzz(1);
     }
 
     @Test
     public void testFuzzSomeDuplicates() throws Exception {
-        testFuzz(1000, 100);
+        testFuzz(100);
     }
 
-    private void testFuzz(int rowCount, int duplicatesPerTick) throws Exception {
+    private void testFuzz(int duplicatesPerTick) throws Exception {
```

Comment:
nit: would be great to assert approx ts values here.

Reply:
it would be hard to add assertion for partition rounding there, not worth it imo. In the test table is partitioned by year, partition cull is unlikely. I would rather have a fix now and think about fuzzing it later (and we can move on with other things)

Reply:
We could at least check that the approx min timestamp is equal or less than the exact min timestamp. Same with the max timestamp.

Reply:
Done in f45c5cff4662338e9621c9d770099d17c195a851